padd.sh

```diff
@@ -959,7 +959,7 @@ SizeChecker(){
         width=53
         height=20
     # Below Regular. Gives you Slim.
-    elif [ "$console_width" -lt "80" ] || [ "$console_height" -lt "22" ]; then
+    elif [ "$console_width" -lt "60" ] || [ "$console_height" -lt "22" ]; then
```

Comment:
You are setting `width=60` if `$console_width` is **less than** 60.

If the console width are 58, this will be true and the screen will break because PADD will be 60 columns wide.

Reply:
No. The code can actually never get here as the `[ "$console_width" -lt "60" ]` will fire in the `elif` immediately above. I just left it there as this matches the overall style of this function.